docs/standard/base-types/type-conversion.md

```diff
@@ -1,7 +1,7 @@
 ---
-title: "Type Conversion in the .NET Framework"
+title: "Type Conversion in .NET"
 description: Read about type conversion in .NET, which creates a value in a new type that's equivalent to the old type's value, but may not keep the original's identity.
-ms.date: "03/30/2017"
+ms.date: 10/22/2020
```

Comment:
nit. Personally I don't think we should update the date unless we're doing a full review on the article. Changing the date indicates that the article was validated for accuracy, but I don't think that was the point of your changes.